.github/workflows/wiki_push.yml

```diff
@@ -0,0 +1,34 @@
+#
+# Workflow to push changes from the "wiki" folder in the repository towards the actual wiki itself.
+#
+name: 'Push to wiki'
+
+on:
+  push:
+    #
+    # Only push when changed files are in the wiki/ folder and on the master branch.
+    #
+    paths:
+    - 'wiki/**'
+    branches:
+    - 'master'
+    #
+    # Ignore any release to avoid accidental trigger of this action.
+    #
+    tags-ignore:
+    - '**'
```

Comment:
Is this even needed if you check for `wiki/**` anyway?

Reply:
yes, because a release causes a push event of the entire repository. Had this many times before.